Controllers/UsersController.cs

```diff
@@ -51,5 +50,25 @@ public async Task<IActionResult> CreateUserAsync([FromBody] UserCredentialResour
             var userResource = _mapper.Map<User, UserResource>(response.User);
             return Ok(userResource);
         }
+
+        [HttpPut("{id}")]
+        [Authorize(Roles = ("Administrator"))]
+        public async Task<IActionResult> ChangeUserPassword([FromBody] UpdateUserPasswordResource
+            updateUserPasswordResource)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
+
+            var user = await _userService.FindByEmailAsync(updateUserPasswordResource.Email);
+            var response = await _userService.ChangeUserPasswordAsync(user, updateUserPasswordResource.Password);
+            if (!response.Success)
+            {
+                return BadRequest(response.Message);
+            }
+
+            return Ok("Passordet er nå endret!");
```

Comment:
Du får en response på linje 65. DU har jo allerede laga en beskjed i service. Kan du ikke bruk den? Helst returner den responsen.

Reply:
not false